R/helpers.R

```diff
@@ -29,42 +49,9 @@ model_file_dict <- function(run_id = NULL, year = NULL) {
   return(dict)
 }
 
-# Get a vector of S3 paths to the artifacts for a given model run
-model_get_s3_artifacts_for_run <- function(run_id, year) {
-  # Get paths of all run objects based on the file dictionary
-  paths <- model_file_dict(run_id, year)
-  s3_objs <- grep("s3://", unlist(paths), value = TRUE)
-  bucket <- strsplit(s3_objs[1], "/")[[1]][3]
-
-  # First get anything partitioned only by year
-  s3_objs_limited <- grep(
-    ".parquet$|.zip$|.rds|.html$", s3_objs,
-    value = TRUE
-  ) %>%
-    unname()
-
-  # Next get the prefix of anything partitioned by year and run_id
-  s3_objs_dir_path <- file.path(
-    grep(
-      ".parquet$|.zip$|.rds$",
-      s3_objs,
-      value = TRUE, invert = TRUE
-    ),
-    glue::glue("year={year}"),
-    glue::glue("run_id={run_id}")
-  )
-  s3_objs_dir_path <- gsub(paste0("s3://", bucket, "/"), "", s3_objs_dir_path)
-  s3_objs_dir_path <- gsub("//", "/", s3_objs_dir_path)
-  s3_objs_w_run_id <- s3_objs_dir_path %>%
-    purrr::map(~ aws.s3::get_bucket_df(bucket, .x)$Key) %>%
-    unlist() %>%
-    purrr::map_chr(~ glue::glue("s3://{bucket}/{.x}"))
-
-  return(c(s3_objs_limited, s3_objs_w_run_id))
-}
-
```

Comment:
The function gets used by model manipulation helpers (tag, delete) that weren't imported to this repo, so no need to include it.